afs/afs-ws/afs-ws-client-utils/src/main/java/com/powsybl/afs/ws/client/utils/RemoteServiceConfig.java

```diff
@@ -27,6 +27,14 @@
 
     private boolean secure;
 
+    private boolean autoreconnect;
+
+    private int reconnectionInitialInterval;
+
+    private int reconnectionIntervalMutiplier;
+
+    private int reconnectionMaxInterval;
+
     public RemoteServiceConfig(String hostName, String appName, int port, boolean secure) {
```

Comment:
Add javadoc to all new methods to at least specify the units. Add a small description to this class to explain how do you use the new fields.

Reply:
it is done 
https://github.com/po605-AFS-listener-reconnection-handlingwsybl/powsybl.github.io/pull/45/files